templates/index.html

```diff
@@ -131,8 +148,9 @@ <h3>Videos</h3>
     url: '{{ url_for('upload') }}',
     paramName: "file", // The name that will be used to transfer the file
     acceptedFiles: 'video/*',
+    parallelUploads: 1,
```

Comment:
What is the reason for this addition?

Reply:
There was a bug in that if the upload function in app.py was called by multiple threads, lines 141-143 in app.py would be executed resulting in overwriting session['jobs'] to be an empty list for each upload (therefore you would only see the last job uploaded). I tried resolving this by making a semaphore for the critical section (c.s. on lines 141-146 in app.py) but that for some reason did not work (somehow separate threads were not notified if a session was updated, even after the session.modified=True line on , so I resulted to uploading files sequentially so as to not run into this problem. I can write a comment here with a condensed explanation

Reply:
Please do!

Reply:
https://github.com/MLDSAI/VisionTracing/pull/2/commits/1a92552fc21b803e122b4fddb207e2b3d57f3674